src/libraries/System.IO.Hashing/src/System.IO.Hashing.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Are you intentionally not adding a .NETCoreApp configuration?

Reply:
I recommend to add one as it avoids the need of the netstandard shim on .NETCoreApp.

Reply:
My thoughts were mainly about keeping the package size small.  But, OK, I'll add a net6.0 configuration (and use netcoreapp3.1 to test the .NET Standard 2.0 implementation).

Reply:
Unfortunately I don't think there is infrastructure on CI to test for older .NETCoreApp versions. If you think local validation is fine, then feel free to add a .NETCoreApp configuration.

Reply:
OK, "have faith that it's the same as the netfx build, but with different dependencies, and that compile can be equated to success" it is, I guess :smirk: